plugins/document/firestore/firestore_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021 Nitric Pty Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package firestore_service_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"syscall"
+
+	"cloud.google.com/go/firestore"
+	ds_plugin "github.com/nitric-dev/membrane/plugins/document/firestore"
+	"github.com/nitric-dev/membrane/plugins/document/test"
```

Comment:
I don't have a better suggestion on where to put this module right now, but placing it in parallel with the plugin implementations feels a bit odd.

Reply:
do you mean this being an integration test, feels a bit odd to have this here?